api/api.go

```diff
@@ -106,7 +108,12 @@ func (w Wrapper) ConnectCustomer(ctx echo.Context) error {
 		town = *connectReq.Town
 	}
 
-	customer, err := w.CustomerService.ConnectCustomer(connectReq.Id, connectReq.Name, town)
+	spID, err := w.SPService.Repository.Get()
+	if err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest, fmt.Errorf("unable to fetch service provider ID: %w", err))
```

Comment:
Why fetch just the ID from the bolt? I think `Repository` isn't meant to be exposed (but it was convenient for creating `SPService`). Better call `SPService.Get()` then you know for sure the service provider DID document is resolvable on the network.